src/class-parsely.php

```diff
@@ -359,6 +360,16 @@ public function get_options(): array {
 			return $this->option_defaults;
 		}
 
+		/**
+		 * Makes option `site_id` compatible with `apikey`.
+		 *
+		 * @since 3.7.0
+		 * @todo Should be removed after 4.0.0 upgrade
+		 */
+		if ( isset( $options['apikey'] ) ) {
+			$options['site_id'] = $options['apikey'];
```

Comment:
I would probably just avoid changing the name of the option to navigate the upgrade/downgrade issues Alex mentioned. You can put a nice, long comment here that references `site_id` so that its clear and searchable.

Reply:
Apologies didn't get the suggestion can you please explain it a bit. There are multiple approaches to do this task so please let me know what you are suggesting.

1. Don't do anything and keep the code as it is.
2. Change all functions and all occurrances of `API Key` string but don't change the option key.
3. Change everything possible with backward compatibility.

I have gone with 3 and I think you are suggesting 2 ..... right? @acicovic please also share your thoughts on it. Thanks

Reply:
I think that @chriszarate is suggesting 2, the reason being that it is guaranteed not to break. With 3, we're introducing a chance of breakage even if our code is good. My preference for stability is 2. My preference for the "most updated code" is 3.

I've encountered an edge case where this can break, and will be posting it about below. Given we decide to go to solution 2, would the change need to happen in many places?

Reply:
@acicovic changes will be done in couple of places and I can do it.

So updating the PR with suggested change and will let you know when it is ready for review again.